src/current/_includes/common/upgrade/upgrade-high-level.md

```diff
@@ -5,8 +5,12 @@ A major-version upgrade involves the following high-level steps, which are descr
 {% endif %}
 
 1. On one node at a time:
```

Comment:
Is it possible to scope this whole section to product tier. The mention of nodes isn't relevant to Standard.

Reply:
Resolved. Removed this include from the main cloud upgrade doc, and improved other tier specific upgrade details, to cover this.